src/main/scala/uk/gov/hmrc/play/microservice/bootstrap/JsonErrorHandling.scala

```diff
@@ -32,18 +32,24 @@ trait JsonErrorHandling extends GlobalSettings {
   implicit val erFormats = Json.format[ErrorResponse]
 
   override def onError(request: RequestHeader, ex: Throwable) = {
-    Future.successful {
-      val (code, message) = ex match {
-        case e: HttpException => (e.responseCode, e.getMessage)
-
-        case e: Upstream4xxResponse => (e.reportAs, e.getMessage)
-        case e: Upstream5xxResponse => (e.reportAs, e.getMessage)
+    logError(request, ex)
+    Future.successful(resolveError(ex))
+  }
 
-        case e: Throwable => (INTERNAL_SERVER_ERROR, e.getMessage)
-      }
+  private def logError(request: RequestHeader, ex: Throwable): Unit = {
```

Comment:
Now that exception handling has been removed (as it was unnecessary), can this `logError` be inlined in `onError`? It looks like there's a single statement here.